src/convertToObject.js

```diff
@@ -7,6 +7,21 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.includes(':'))
+    .reduce((obj, item) => {
+      const [property, ...value] = item.split(':');
+      const trimProperty = property.trim();
+      const trimValue = value.join(':').trim();
```

Comment:
The `trimValue` variable is also correctly defined and used; this ensures that the values in the resulting object do not contain any leading or trailing whitespace, which is crucial for accurate CSS representation.